qdrant_client/migrate/migrate.py

```diff
@@ -115,15 +131,31 @@ def _migrate_collection(
         dest_client (QdrantBase): Destination client
         batch_size (int, optional): Batch size for scrolling and uploading vectors. Defaults to 100.
     """
-    records, next_offset = source_client.scroll(collection_name, limit=2, with_vectors=True)
-    dest_client.upload_records(collection_name, records)
-    while next_offset is not None:
+    next_offset = None
+    while True:
         records, next_offset = source_client.scroll(
-            collection_name, offset=next_offset, limit=batch_size, with_vectors=True
+            collection_name=collection_name,
+            offset=next_offset,
+            limit=batch_size,
+            with_vectors=True,
+        )
+        if not records:
+            break
+        forgive_exception(
+            func=dest_client.upload_records,
+            collection_name=collection_name,
+            records=records,
+            wait=True,
         )
-        dest_client.upload_records(collection_name, records, wait=True)
-    source_client_vectors_count = source_client.get_collection(collection_name).vectors_count
-    dest_client_vectors_count = dest_client.get_collection(collection_name).vectors_count
```

Comment:
let's remove forgive_exception and just implement it in-place, cuz the function is not really elegant

Reply:
for all kind of exceptions or there's specific exception that occurs ?